src/log-imported-targets.ts

```diff
@@ -35,7 +38,7 @@ export async function logImportedTarget(
     });
     debug(
       {
-        target,
+        target: _.pick(target, ...targetProps),
```

Comment:
this ignores any other "meta" that SCM could have returned like "forked" and "archived" as it is not interesting for the purpose of the comparisons and won't be ever present in Snyk